.github/workflows/ci.yml

```diff
@@ -46,7 +46,7 @@ jobs:
               with:
                   # Checkout gally "template" from the same branch. This is the dev environment with docker files.
                   repository: 'Elastic-Suite/gally'
-                  ref: 1.0.x # ${{ github.base_ref }} cannot work on pushes.
+                  ref: master # ${{ github.base_ref }} cannot work on pushes.
```

Comment:
Beware, this should not move in 1.0.x

Reply:
Ah yes, I build my branch from the wrong ref